frontend/docsettings.lua

```diff
@@ -134,6 +149,43 @@ function DocSettings:getFileFromHistory(hist_name)
     end
 end
 
+--- Returns path to book custom cover file if it exists, or nil.
+function DocSettings:findCoverFile(doc_path)
+    local function findCoverFileInDir(dir)
```

Comment:
How often is findCoverFile() called? If often, this local function will be created each time.
I would not find it being another DocSettings function, just before this one, named `DocSettings:_findCoverFile(dir)` to indicate it's somehow a helper for the function just after.